### PR TITLE
Add finnish translation for linux distributions

### PR DESCRIPTION
**Pull request by larevehkaoja**

### CODE REVIEW

File: dist/linux/info.cemu.Cemu.metainfo.xml [hunk 1]
```diff
@@ -40,6 +43,8 @@
     <p xml:lang="es">Fue escrito desde cero y el desarrollo del proyecto comenzó aproximadamente a principios de 2015.</p>
     <p xml:lang="pt_BR">Foi escrito do zero e o desenvolvimento do projeto começou aproximadamente a princípio de 2015.</p>
     <p xml:lang="it">È stato scritto da zero e lo sviluppo del progetto è iniziato circa all'inizio del 2015.</p>
+    <p xml:lang="fi"Se kirjoitettiin tyhjästä ja sen kehitys alkoi suunnilleen vuoden 2015 alussa.</p>
+
```
Comment (JakobDev) [inline dist/linux/info.cemu.Cemu.metainfo.xml:47]:
```suggestion
    <p xml:lang="fi"Se kirjoitettiin tyhjästä ja sen kehitys alkoi suunnilleen vuoden 2015 alussa.</p>
```

**Comment by marcoluc97** [comment]
As Jakob said update also this file:

_dist/linux/info.cemu.Cemu.desktop_

**Comment by larevehkaoja** [comment]
Now I updated _dist/linux/info.cemu.Cemu.desktop_ as well

File: dist/linux/info.cemu.Cemu.metainfo.xml [hunk 2]
```diff
@@ -40,6 +43,8 @@
     <p xml:lang="es">Fue escrito desde cero y el desarrollo del proyecto comenzó aproximadamente a principios de 2015.</p>
     <p xml:lang="pt_BR">Foi escrito do zero e o desenvolvimento do projeto começou aproximadamente a princípio de 2015.</p>
     <p xml:lang="it">È stato scritto da zero e lo sviluppo del progetto è iniziato circa all'inizio del 2015.</p>
+    <p xml:lang="fi"Se kirjoitettiin tyhjästä ja sen kehitys alkoi suunnilleen vuoden 2015 alussa.</p>
```
Comment (Exzap) [inline dist/linux/info.cemu.Cemu.metainfo.xml:46]:
```suggestion
    <p xml:lang="fi">Se kirjoitettiin tyhjästä ja sen kehitys alkoi suunnilleen vuoden 2015 alussa.</p>
```

**Comment by Exzap** [comment]
thanks!

**Pull request merged**